virt/vm_networking/virt-networking-overview.adoc

```diff
@@ -60,6 +60,15 @@ ifdef::openshift-rosa,openshift-dedicated[]
 You can expose a VM within the cluster or outside the cluster by creating a `Service` object.
 endif::openshift-rosa,openshift-dedicated[]
 
+[id="primary-udn-config"]
+== Configuring a primary user-defined network
+
+xref:../../virt/vm_networking/virt-connecting-vm-to-primary-udn.adoc#virt-connecting-vm-to-primary-udn[Connecting a virtual machine to a primary user-defined network]::
+
+You can connect a virtual machine (VM) to a user-defined network (UDN) on the VM’s primary interface. The primary user-defined network replaces the default pod network to connect pods and VMs in selected namespaces. Tenant owners can configure a primary `UserDefinedNetwork` CRD to create a network that isolates their namespace from other namespaces without requiring network policies. Cluster administrators can use the `ClusterUserDefinedNetwork` CRD to create a shared OVN `layer2` network across multiple namespaces.
++
+User-defined networks with the `layer2` overlay topology are a good fit for VM workloads, and a good alternative to secondary networks in environments where physical network access is limited, such as the public cloud. The `layer2` topology enables seamless migration of VMs without the need for Network Address Translation (NAT) because persistent IP addresses are preserved across cluster nodes during live migration.
```

Comment:
"Good fit" seems like marketing parlance. How about:

```suggestion
User-defined networks with the `layer2` overlay topology are useful for VM workloads, and a good alternative to secondary networks in environments where physical network access is limited, such as the public cloud. The `layer2` topology enables seamless migration of VMs without the need for Network Address Translation (NAT) because persistent IP addresses are preserved across cluster nodes during live migration.
```